cmd/resim/commands/sweep.go

```diff
@@ -56,6 +56,7 @@ const (
 	sweepParameterValuesKey  = "parameter-values"
 	sweepExitStatusKey       = "exit-status"
 	sweepGithubKey           = "github"
+	sweepAccountKey          = "sweep"
```

Comment:
Is this string the actual flag? i.e. `./resim sweep create --sweep my-github-handle`? Should it be `--account` instead?

Reply:
gah - that's why the test is failing :)